stytch/consumer/api/m2m.py

```diff
@@ -26,4 +30,113 @@ def __init__(
         self.sync_client = sync_client
         self.async_client = async_client
         self.jwks_client = jwks_client
+        self.project_id = project_id
         self.clients = Clients(api_base, sync_client, async_client)
+
+    # MANUAL(m2m.token)(SERVICE_METHOD)
+    # ADDIMPORT: from typing import Any, Dict, List
+    # ADDIMPORT: from stytch.consumer.models.m2m import GetTokenResponse
+    def token(
+        self,
+        client_id: str,
+        client_secret: str,
+        scopes: Optional[List[str]] = None,
+    ) -> GetTokenResponse:
+        """Retrieves an access token for the given M2M Client.
+        Access tokens are JWTs signed with the project's JWKs, and are valid for one hour after issuance.
+        M2M Access tokens contain a standard set of claims as well as any custom claims generated from templates.
+
+        Fields:
+          - client_id: The ID of the client.
+          - client_secret: The secret of the client.
+          - scopes: An array scopes requested. If omitted, all scopes assigned to the client will be returned.
```

Comment:
[sand] missing `of` in description
```suggestion
          - scopes: An array of scopes requested. If omitted, all scopes assigned to the client will be returned.
```